test/detectors/SendInLoop.expected.out

```diff
@@ -1,62 +1,53 @@
+  6 |                 to: sender(),
 Help: Consider refactoring to avoid calling send functions inside loops
 See: https://nowarp.io/tools/misti/docs/detectors/SendInLoop
 
 [MEDIUM] SendInLoop: Send function called inside a loop
-test/detectors/SendInLoop.tact:16:13:
-  15 |         repeat (10) {
-> 16 |             send(SendParameters{
+test/detectors/SendInLoop.tact:15:13:
+  14 |         repeat (10) {
+> 15 |             send(SendParameters{
                    ^
-  17 |                 to: sender(),
+  16 |                 to: sender(),
 Help: Consider refactoring to avoid calling send functions inside loops
 See: https://nowarp.io/tools/misti/docs/detectors/SendInLoop
 
 [MEDIUM] SendInLoop: Send function called inside a loop
-test/detectors/SendInLoop.tact:26:13:
-  25 |         do {
-> 26 |             send(SendParameters{
+test/detectors/SendInLoop.tact:25:13:
+  24 |         do {
+> 25 |             send(SendParameters{
                    ^
-  27 |                 to: sender(),
+  26 |                 to: sender(),
 Help: Consider refactoring to avoid calling send functions inside loops
 See: https://nowarp.io/tools/misti/docs/detectors/SendInLoop
 
 [MEDIUM] SendInLoop: Send function called inside a loop
-test/detectors/SendInLoop.tact:37:13:
-  36 |         foreach (k, v in m) {
-> 37 |             send(SendParameters{
+test/detectors/SendInLoop.tact:36:13:
+  35 |         foreach (k, v in m) {
+> 36 |             send(SendParameters{
                    ^
-  38 |                 to: sender(),
+  37 |                 to: sender(),
 Help: Consider refactoring to avoid calling send functions inside loops
 See: https://nowarp.io/tools/misti/docs/detectors/SendInLoop
 
 [MEDIUM] SendInLoop: Send function called inside a loop
-test/detectors/SendInLoop.tact:48:17:
-  47 |             repeat (10) {
-> 48 |                 send(SendParameters{
+test/detectors/SendInLoop.tact:47:17:
+  46 |             repeat (10) {
+> 47 |                 send(SendParameters{
                        ^
-  49 |                     to: sender(),
+  48 |                     to: sender(),
 Help: Consider refactoring to avoid calling send functions inside loops
 See: https://nowarp.io/tools/misti/docs/detectors/SendInLoop
 
-[MEDIUM] SendInLoop: Send function called inside a loop
-test/detectors/SendInLoop.tact:80:19:
-  79 |             i += 1;
-> 80 |          let a =  send(SendParameters{
-                         ^
-  81 |                 to: self.owner,
-Help: Consider refactoring to avoid calling send functions inside loops
-See: https://nowarp.io/tools/misti/docs/detectors/SendInLoop
-
-[MEDIUM] SendInLoop: Send function called inside a loop
-test/detectors/SendInLoop.tact:93:13:
-  92 |         while (i < 5) {
-> 93 |             self.reply(Msg{ a: i }.toCell());
-                   ^
-  94 |             i = i + 1;
+[MEDIUM] SendInLoop: Function "ComplexIndirectSend::deepNestSend" called inside a loop leads to a send function
```

Comment:
It looks strange. Let's write that't a method or simply put the function name here.

Reply:
done